src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -611,6 +611,10 @@ Emscripten_HandleResize(int eventType, const EmscriptenUiEvent *uiEvent, void *u
                 emscripten_set_element_css_size(NULL, w, h);
             }
 
+            /* force the event to trigger, so pixel ratio changes can be handled */
```

Comment:
We should probably only do this if the ratio has changed.

Reply:
Won't that be already handled by the browser? (as in, won't this code be executed only if there's actual change of the window size or pixel ratio?)

Reply:
It's possible for the canvas size to not change when the window resizes (min/max width, or even the default where the canvas doesn't change size).

Reply:
Ah, right, the handler's attached to a window... Pity that `ResizeObserver` is still Chrome-only :( OK then, I'll update the patch.